binder/start

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+#
+# Copyright 2018-2020 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import sys
+import shutil
+import os
+
+# Add path to notebook dir
+args = sys.argv + ['--notebook-dir', './binder-demo']
+
+print(args, flush=True)
+os.execv(args)
```

Comment:
I don't understand how `sys.argv` becomes `jupyter lab ...` in any of this.  How/where does binder get the notion that that's the target command?  Is there some configuration somewhere besides `postBuild` and `start`?

Reply:
removed this portion...this was accidental push to the pr branch instead of another working branch i had